<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Abbreviation</class>
 <widget class="QMainWindow" name="Abbreviation">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Abbreviation</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,5">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEdit_search"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_6">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea_abbreDisplay">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>222</width>
             <height>488</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_abbreButton">
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
           <zorder>verticalSpacer</zorder>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser_main">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>816</width>
     <height>23</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_content">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="action_set_content"/>
   </widget>
   <widget class="QMenu" name="menu_test">
    <property name="title">
     <string>测试</string>
    </property>
    <addaction name="action_add_button"/>
    <addaction name="action_text_show"/>
    <addaction name="action_fileProTest"/>
    <addaction name="action_test"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_content"/>
   <addaction name="menu_help"/>
   <addaction name="menu_test"/>
  </widget>
  <action name="action_add_button">
   <property name="text">
    <string>新增按钮</string>
   </property>
  </action>
  <action name="action_text_show">
   <property name="text">
    <string>文本显示</string>
   </property>
  </action>
  <action name="action_set_content">
   <property name="text">
    <string>设置目录</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>扫描</string>
   </property>
  </action>
  <action name="action_fileProTest">
   <property name="text">
    <string>文本处理测试</string>
   </property>
  </action>
  <action name="action_test">
   <property name="text">
    <string>测试</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
